### PR TITLE
test: Add Node 20 to CI matrix

### PR DESCRIPTION
**Pull request by BilalQamar95**
### Description

As a first step in the upgrade to Node 20, add it to the CI matrix as a non-blocking test.

See [the tracking issue](https://github.com/openedx/frontend-app-learning/issues/1316) for further information.

### CODE REVIEW

File: src/courseware/course/content-tools/calculator/Calculator.jsx [hunk 1]
```diff
@@ -341,6 +341,7 @@ class Calculator extends Component {
                         <li>arccsch(4x+y)</li>
                       </ul>
                     </td>
+                    { /* eslint-disable-next-line jsx-a11y/control-has-associated-label */ }
```
Comment (UsamaSadiq) [inline src/courseware/course/content-tools/calculator/Calculator.jsx]: Let's mention why this change is needed.

Comment (BilalQamar95) [reply src/courseware/course/content-tools/calculator/Calculator.jsx]: The error `(A control must be associated with a text label)` and the corresponding ESLint ignore might seem self-explanatory as the rule `jsx-a11y/control-has-associated-label` typically applies to interactive controls like buttons, inputs, or other form elements that require labels for accessibility. In this case, `<td>` is a table cell element, which doesn’t function as an interactive control and therefore doesn’t require a label. The `eslint-disable` comment was added to prevent unnecessary warnings for this specific use case.

**Review by UsamaSadiq — APPROVED** [review]

**Review by brian-smith-tcril — APPROVED** [review]
